pgBouncer 1.25.0 fails to build on ppc64le

Hi,

I'm getting the following errors while trying to build 1.25.0 on ppc64le (RHEL 8, 9 and 10). The other archs are just fine.

Can you please take a look? Thanks!
Devrim

```
gcc  -DHAVE_CONFIG_H -Iinclude -Ilib   -O2 -flto=auto -ffat-lto-objects -fexceptions -g -grecord-gcc-switches -pipe -Wall -Werror=format-security -Wp,-U_FORTIFY_SOURCE,-D_FORTIFY_SOURCE=3 -Wp,-D_GLIBCXX_ASSERTIONS -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -fstack-protector-strong -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1  -m64 -mcpu=power9 -mtune=power10 -fasynchronous-unwind-tables -fstack-clash-protection  -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -Wmissing-prototypes -Wpointer-arith -Wendif-labels -Wdeclaration-after-statement -Wold-style-definition -Wstrict-prototypes -Wundef -Wformat=2 -Wuninitialized -Wmissing-format-attribute -MD -MP -MT .objs/pgbouncer/lib/usual/time.o -MF .objs/pgbouncer/lib/usual/time.o.d -c -o .objs/pgbouncer/lib/usual/time.o lib/usual/time.c
In file included from include/common/ascii.h:14,
                 from src/common/wchar.c:22:
include/common/simd.h: In function ‘vector8_has_le’:
include/common/simd.h:239:14: error: ‘int64’ undeclared (first use in this function); did you mean ‘uint64’?
  239 |         if ((int64) v >= 0 && c < 0x80)
      |              ^~~~~
      |              uint64
include/common/simd.h:239:14: note: each undeclared identifier is reported only once for each function it appears in
include/common/simd.h:239:20: error: expected ‘)’ before ‘v’
  239 |         if ((int64) v >= 0 && c < 0x80)
      |            ~       ^~
      |                    )
gcc  -DHAVE_CONFIG_H -Iinclude -Ilib   -O2 -flto=auto -ffat-lto-objects -fexceptions -g -grecord-gcc-switches -pipe -Wall -Werror=format-security -Wp,-U_FORTIFY_SOURCE,-D_FORTIFY_SOURCE=3 -Wp,-D_GLIBCXX_ASSERTIONS -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -fstack-protector-strong -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1  -m64 -mcpu=power9 -mtune=power10 -fasynchronous-unwind-tables -fstack-clash-protection  -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -Wmissing-prototypes -Wpointer-arith -Wendif-labels -Wdeclaration-after-statement -Wold-style-definition -Wstrict-prototypes -Wundef -Wformat=2 -Wuninitialized -Wmissing-format-attribute -MD -MP -MT .objs/pgbouncer/lib/usual/tls/tls.o -MF .objs/pgbouncer/lib/usual/tls/tls.o.d -c -o .objs/pgbouncer/lib/usual/tls/tls.o lib/usual/tls/tls.c
make[1]: *** [/var/lib/pgsql/rpmcommon/BUILD/pgbouncer-1.25.0/lib/mk/antimake.mk:1233: .objs/pgbouncer/src/common/wchar.o] Error 1
```

--- a/include/common/postgres_compat.h
+++ b/include/common/postgres_compat.h
@@ -17,6 +17,9 @@
 #endif
 
 #define int8 int8_t
+#define int16 int16_t
+#define int32 int32_t
+#define int64 int64_t
 #define uint8 uint8_t
 #define uint16 uint16_t
 #define uint32 uint32_t

